radar_control/radar_control.py

```diff
@@ -613,30 +612,28 @@ def radar():
                         beamdir = slice_to_beamdir_dict[slice_id]
                         beam_phase_dict[slice_id] = \
                             rx_azimuth_to_antenna_offset(beamdir, options.main_antenna_count,
-                                                      options.interferometer_antenna_count,
-                                                      options.main_antenna_spacing,
-                                                      options.interferometer_antenna_spacing,
-                                                      options.intf_offset, receive_freq)
+                                                         options.interferometer_antenna_count,
+                                                         options.main_antenna_spacing,
+                                                         options.interferometer_antenna_spacing,
+                                                         options.intf_offset, receive_freq)
 
                     beam_phase_dict_list.append(beam_phase_dict)
 
-
                 # Setup debug samples if in debug mode.
                 debug_samples = []
                 if __debug__:
                     for sequence_index, sequence in enumerate(aveperiod.sequences):
                         sequence_samples_dict = create_debug_sequence_samples(experiment.txrate,
-                                              experiment.txctrfreq,
-                                              sequence_dict_list[sequence_index],
-                                              options.main_antenna_count,
-                                              experiment.output_rx_rate,
-                                              sequence.ssdelay)
+                                                              experiment.txctrfreq,
+                                                              sequence_dict_list[sequence_index],
+                                                              options.main_antenna_count,
+                                                              experiment.output_rx_rate,
+                                                              sequence.ssdelay)
                         debug_samples.append(sequence_samples_dict)
 
-                # all phases are set up for this averaging period for the beams required. Time to start averaging
-                # in the below loop.
-                num_sequences = 0
-                time_remains = True
+                # all phases are set up for this averaging period for the beams required.
+                # Time to start averaging in the below loop.
```

Comment:
good idea to move these initializations closer to the loop. I would remove the comment here too since it applies to the loop further below
```suggestion

```